tests/component-tests/textToSpeech/js/TextToSpeechTests.js

```diff
@@ -11,13 +11,40 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 */
 
 // Declare dependencies
-/* global fluid, jqUnit, speechSynthesis */
+/* global fluid, jqUnit, speechSynthesis, SpeechSynthesisUtterance */
 
 (function () {
     "use strict";
 
     fluid.registerNamespace("fluid.tests");
 
+    // Ensures that TTS is supported in the browser, including cases wehre the
+    // feature is detected, but where the underlying audio engine is missing.
+    // For example VM's on SauceLabs.
+    //
+    // returns a promise
```

Comment:
I think that i) this code provides a feature detection which is generally useful, and could well be useful outside just the testing scenario, and that ii) it should be moved out of the testing files into the main implementation, and iii) its documentation should be improved